src/index.js

```diff
@@ -1 +1,9 @@
 'use strict';
+
+const app = require('./app.js');
+
+require('dotenv').config();
+
+const PORT = process.env.PORT || 3005;
+
+app.listen(PORT, () => {});
```

Comment:
Consider adding a console log inside the callback function to confirm that the server is running, e.g., `console.log(`Server is running on port ${PORT}`);`.